### PR TITLE
threads for clients - added connection retrying

### PR DESCRIPTION
**Pull request by rowanvb**
mooi dit

### CODE REVIEW

**Comment by leonkielstra** [comment]
🙈

**Comment by Feikoritsema** [comment]
@leonkielstra 💯

File: hub/src/main/java/hub/Hub.java [hunk 1]
```diff
@@ -113,33 +120,37 @@ public synchronized void setStatus(final Status s) {
...
     }
 
     private void initializeEmergencySequence() {
-        System.out.println("Sending message now!");
-        emergencyConnection = new Client();
-        emergencyConnection.connectTo(settings.getEmergencyService(), 4242);
-        EmergencyMessage message = new EmergencyMessage(EmergencyMessage.Action.OPEN, "New Emergency procedure started.");
-        emergencyConnection.send(message);
+        queue = new LinkedBlockingQueue<>();
+        emergencyConnectionHandler = new LastingClientHandler(settings.getEmergencyService(), 4242, queue);
```
Comment (Feikoritsema) [inline hub/src/main/java/hub/Hub.java:135]: > 4242 
- @leonkielstra

**Review by leonkielstra — APPROVED** [review]
😍

**Pull request merged**